app/screens/user_profile/user_profile.tsx

```diff
@@ -54,6 +55,7 @@ const SINGLE_OPTION_HEIGHT = 68;
 const LABEL_HEIGHT = 58;
 const EXTRA_HEIGHT = 60;
 const MANAGE_ICON_HEIGHT = 72;
+const CUSTOM_ATTRIBUTES_HEIGHT = 200; // Height to show approximately 4-5 attributes
```

Comment:
// Height to show approximately 4-5 attributes

This comment, does it apply to iPad, iPhone?

Reply:
I added it for iPhone but now I have to wonder if that applies to iPad, and I'm not sure how that translates from one to another 🤔 

For the current PR I'm leaving it as it is, but we can discuss offline how to do this for the final version of the feature.
Thanks for raising that :)

Reply:
Let's talk about this.

There is a useTablet hooks or something like that, that will give you the device is a tablet vs a phone.

But let's get this through and get another ticket to address the issue (if any).